config/locales/en/pages.yml

```diff
@@ -1,5 +1,31 @@
 en:
   pages:
+    "404":
+      title: This page cannot be found
+      body_html: |
+        <p>You can <a href="/">start a new prison visit request</a> or book
+        your visit by phone instead. See the
+        <a href="http://www.justice.gov.uk/contacts/prison-finder">prison
+        finder</a> for contact details.</p>
+    "500":
+      title: We are experiencing technical difficulties
+      body_html: |
+        <p>The prison visits booking system is not available right now, but
+        we're working hard to get things up and running again.</p>
+
+        <p>You can try again later or book your visit by phone instead. See the
+        <a href="http://www.justice.gov.uk/contacts/prison-finder">prison
+        finder</a> for contact details.</p>
+    "503":
+      title: We are experiencing technical difficulties
+      body_html: |
+        <p>The prison visits booking system is not available as we're carrying
+        out some planned maintenance. You can try again later or book your
+        visit by phone instead.</p>
+
+        <p>You can try again later or book your visit by phone instead. See the
+        <a href="http://www.justice.gov.uk/contacts/prison-finder">prison
+        finder</a> for contact details.</p>
```

Comment:
I think it would be worth splitting these paragraphs out to two fields, and feeding in the Prison Finder link: see [other template](https://github.com/ministryofjustice/prison-visits-2/blob/4c64d9ac6299e036944e07cb7430aa29b9608d95/app/views/feedback_submissions/new.html.erb#L6-L7) for an example.